recipes/arm-variant/meta.yaml

```
@@ -0,0 +1,25 @@
+package:
+  name: arm-variant
+  version: 1.0.0
+
+build:
+  number: 0
+  string: "{{ arm_variant_type }}"
+  skip: true  # [not aarch64]
+  track_features:
+    - arm_{{ arm_variant_type }}  # [arm_variant_type != arm_variant_type_default]
+
+test:
+  commands:
+    - exit 0
+
+about:
+  home: https://github.com/conda-forge/arm-variant-feedstock
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'A meta-package to select the ARM variant.'
```

Can you add a more descriptive summary here?

---

What would you like to see?

---

1. How to use a variant
2. What the difference between sbsa and tegra variant is.
3. How it's used (i.e. that it's used in cudatoolkit)

---

How about this? Also please feel free to edit (if needed) :)

```suggestion
  summary: 'A meta-package to select the ARM variant.'
  description: |
    There are a couple of ARM variants that differ based on architecture.

    Currently the main ones are:
      * SBSA ( Server Base System Architecture ), which is used in clusters, cloud environments, etc.
      * Tegra ( System on a Chip ), which is used in edge applications like mobile internet devices, streaming systems, etc.

    Mainly these differ as they have 2 different `cudatoolkit`s. So this package is used to select between them.

    The default is SBSA. So Tegra users should make sure to change this if needed.
```

---

Can we please move this description to conda-forge.github.io as a documentation effort? IMHO no feedstock maintainer would read the summary/description for things like this. We always refer to the CF docs.

---

We can duplicate it, but please keep it here.

@jakirkham, thanks for adding this.

---

Ofc! Went ahead and committed that change :)